index.js

```diff
@@ -48,9 +48,9 @@ export const EXIT = false
 export const visitParents =
```

Comment:
It seems that TSDoc description only comes through when attached to the const

```suggestion
/**
 * Visit children of tree which pass a test
 */
export const visitParents =
```

Reply:
fixed!